draft-ietf-snac-simple.xml

```diff
@@ -268,30 +268,38 @@
 	  the stub router must provide it.
 	</t>
 	<section>
-	  <name>Usable On-Link Prefixes</name>
+	  <name>Suitable On-Link Prefixes</name>
 	  <t>
-	    IPv6 addressing is considered to be present on the link if a usable on-link prefix is advertised on the AIL.
-	    A usable on-link prefix is a prefix advertised on the link that has a preferred lifetime of 30 minutes
-	    or more, is marked on-link and allows autonomous configuration.</t>
+	    Stub routers must evaluate prefixes that are advertised on-link as to their suitability for use in communicating with
+	    devices on the stub network. If no suitable prefix is found, a stub router MUST advertise one.
+	  </t><t>
+	    An on-link prefix is considered suitable if it is advertised on the link in an RA and has:
+	  </t>
+	  <ul>
+	    <li>a preferred lifetime of 30 minutes or more</li>
+            <li>is marked on-link</li>
+	    <li>has a width of 64 bits (<xref target="RFC7421" section="1" sectionFormat="comma" />), and</li>
+	    <li>allows autonomous configuration.</li>
```

Comment:
```suggestion
	    <li>allows stateless address autoconfiguration (SLAAC) - as indicated by the Autonomous flag being set in the Prefix Information Option.</li>
```

Reply:
Might make sense to put the clarification about PIO in the heading rather than in the last bullet, because all of the conditions that we have to check are actually in the PIO.